src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java

```diff
@@ -688,6 +693,26 @@ private void showView(Profile place) {
 		projectSynAlert.clear();
 		teamSynAlert.clear();
 		String token = place.toToken();
+		if (token.startsWith("message/")) {

```

Comment:
Instead of a generic message, use a set of predefined messages instead.  Look for something like ORC_ID_SUCCESS.